File: src/C#/CreationalPatterns/AbstractFactory/EraStyles/Products/VictorianHouse.cs
using EraStyles.Abstraction;
using EraStyles.Constants;
using System;
using System.Collections.Generic;
using System.Text;

namespace EraStyles.Products
{
    public class VictorianHouse : EraObject
    {
        public override void ShowDetails()
        {
            Console.WriteLine(string.Format(GlobalConstants.HouseMessage, "Victorian Era"));
        }
    }
}

File: src/JavaScript/CreationalPatterns/AbstractFactory/EraStyles/products/victorianHouse.js

const EraObject = require("../abstraction/eraObject");
const GlobalConstants = require("../constants/globalConstants");

class VictorianHouse extends EraObject {
    showDetails() {
        console.log(GlobalConstants.HouseMessage.replace("{0}", "Victorian Era"));
    }
}

module.exports = VictorianHouse;